frontend/cypress/e2e/change-complaint-filters-view.cy.ts

```diff
@@ -24,7 +24,7 @@ describe("Complaint Change Status spec - Details View", () => {
           expect(length, "rows N").to.be.gt(0);
         });
 
-      cy.get("#complaint-filter-image-id").click({ force: true });
```

Comment:
this might be my only gripe, and its not a huge issue, more of a constancy thing to avoid colliding with existing styles while you were refactoring things, but this ```#complaint-filter-image-id``` should be renamed to ```#comp-filter-image-id```. This way its inline with our other styles

Reply:
Looks like I missed updating that test, however I've renamed this to #comp-filter-btn".